src/black/brackets.py

```diff
@@ -49,7 +49,9 @@
 DOT_PRIORITY: Final = 1
 
 
-class BracketMatchError(KeyError):
+# Ideally this would be a subclass of KeyError, but mypyc doesn't like that.
```

Comment:
I don't think it really needs to be a subclass of KeyError. I say we just remove this comment and leave it a direct subclass of Exception.

Reply:
Sounds good to me!